app/Policies/OfferPolicy.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace App\Policies;
+
+use App\Models\Offer;
+use App\Models\User;
+use Illuminate\Auth\Access\Response;
+
+class OfferPolicy
+{
+    /**
+     * Determine whether the user can view any models.
+     */
+    public function viewAny(User $user): bool
+    {
+        // ograć, jak będą role
+        return false;
```

Comment:
Jśli dobrze kojarzę to OfferController zwraca tylko oferty dla zalogowanego użytkownika, więc tutaj można zrobić po prostu true. A dostęp do konkretnych ofert ograć na zasadzie global scope

Reply:
Hmm, a jest sens w takim razie w ogóle to zostawiac jeśli ma być true?